### PR TITLE
Issue #848 - improve layout CSSFixme

### PR DESCRIPTION
**Pull request by magsout**
Added some improvement:
 - https://github.com/webcompat/webcompat.com/pull/887#issuecomment-167939419
 - https://github.com/webcompat/webcompat.com/pull/887#commitcomment-15193558

capture d ecran 2015-12-30 a 12 51 19

r? @karlcow @hallvors

### CODE REVIEW

**Comment by hallvors** [comment]
@magsout what does code that has been modified look like? See http://hallvord.com/temp/moz/cssfixme.php?url=http%3A%2F%2Fhallvord.com%2Ftemp%2Fmoz%2FcompatTesterTesting%2Fstyle.css for an example.

**Comment by hallvors** [comment]
Very minor language point: I wonder if "CSS code" will be easier to understand than "CSS instructions"..?

**Comment by magsout** [comment]
@hallvors aaaaahh, ok, :-1: my bad... So, I return to the previous style for the `<pre>`, and I only highlights the strong elements

**Comment by magsout** [comment]
capture d ecran 2015-12-30 a 14 24 07
capture d ecran 2015-12-30 a 14 24 13


@hallvors @karlcow @miketaylr Which one do you prefer ?

**Comment by karlcow** [comment]
@magsout The 1st one (at the top). Many thanks. Super travail!

**Comment by magsout** [comment]
@hallvors is it possible to detect if there is content in the `<pre>` element? If so, you could add a class `is-active` because by default, when there is no content we still see the style of the <pre> because of padding (which I can delete it, but it's less pretty)

**Comment by hallvors** [comment]
We can certainly set an is-active style from the JS when inserting the generated code. Or just generate the PRE element too on the fly..? You can just add a DIV id=result-parent or something to put the generated PRE inside, I think that may be cleaner than toggling PRE style based on a class :)

**Comment by hallvors** [comment]
(I also vote for the 1st one - much of the point is to show very clearly what we're changing and it's more obvious that way)

**Comment by magsout** [comment]
>Or just generate the PRE element too on the fly..? You can just add a DIV id=result-parent or something to put the generated PRE inside, I think that may be cleaner than toggling PRE style based on a class :)

@hallvors looks good :+1:

File: webcompat/templates/cssfixme.html [hunk 1]
```diff
@@ -20,7 +28,13 @@
           <button type="button" class="Button Button--default" id="btn_do_fixup">Add standard equivalents for prefixed CSS</button>
         </div>
         <div class="wc-CSSFixme-item">
-          <pre class="wc-CSSFixme-resultat" id="fixedcss"></pre>
+          <div id="js-CSSFixme"></div>
```
Comment (magsout) [inline webcompat/templates/cssfixme.html]: @hallvors I removed `<pre>`  element and add this parent. So you can add <pre> directly in this div. if you can add a class : `class="wc-CSSFixme-resultat"` would be awesome ;)

**Comment by magsout** [comment]
Ok, updated

**Comment by magsout** [comment]
build passed

**Comment by miketaylr** [comment]
Same, top style looks great.

**Comment by hallvors** [comment]
I sort of liked the way the code in the second screenshot stands out with the yellow background though.. I'm testing the JS changes now, when the background is plain it is too much a part of the page, somehow. What about adding for example 

```
.wc-CSSFixme-resultat{
  border-left: 8px solid var(--wc-background-dark);
  padding-left: 5px;
}
```
To make it stand out a bit more?
/me dabbling in designy stuff - feel free to ignore suggestions that betray my lack of taste :p

**Comment by hallvors** [comment]
@magsout I hope you're happy with me pushing directly here..?

**Comment by magsout** [comment]
@hallvors I added the changes to the style. What makes good

capture d ecran 2015-12-30 a 21 26 31

>@magsout I hope you're happy with me pushing directly here..?

it's fine to like that .. :+1: 

I'm reading your email and try to understand the issue

**Comment by miketaylr** [comment]
> It's only a global because the script for some reason is included twice in the page

I'm about to push a fix for that. I don't understand 100%, but the `extrascripts` template block was included *inside* the `body` template block, and I guess it looped over itself... twice? I don't know. But moving extrascripts outside of body (like all the other templates) fixes it. Pushing fix now.

**Comment by magsout** [comment]
👍 @miketaylr

**Comment by magsout** [comment]
👍 @miketaylr

**Comment by miketaylr** [comment]
:+1: :+1: ^_^

**Comment by magsout** [comment]
oups .. ;)

**Comment by magsout** [comment]
Awesome.

@hallvors or @karlcow if everything looks good;)

**Comment by hallvors** [comment]
Great, now we can undo the global-ness of that variable. If I do that quickly enough perhaps I can make it before Mike merges ;)

**Comment by hallvors** [comment]
and @magsout :+1: to the extra styling - stands out now

**Comment by miketaylr** [comment]
Heh, I'll let you pull the merge trigger @hallvors -- and I can deploy this afternoon as well.

**Comment by miketaylr** [comment]
Great work @magsout and @hallvors!

**Pull request merged**